CONTRIB.md

```diff
@@ -89,10 +94,17 @@ of the P/Invoke method they wrap:
 1. The method has a single out parameter that in a naturally managed API would typically
    serve as the return value, and the P/Invoke method's return value is void or an error code.
 1. A set of methods for enumeration can be wrapped with a helper that exposes an IEnumerable.
+   This is the one known case where the name of the helper method may not coincide with the
```

Comment:
I think that we can be more specific there and suggest a few specific name transformation :
* `Foo` | `GetFoo` -> `EnumerateFoo` (As in `Directory.EnumerateFiles`)
* `Foo` -> `FooAsync`

Essentially when the helper is the equivalent of a marshaling to one of the .Net specific domains (Enumerable, Task) the name might be modified to reflect that but there are few or no reason to create a new name.

If a method seem to require a new name there is a big chance that it's too specific for the library.